pkg/analyze/kube_resource.go

```diff
@@ -246,12 +246,14 @@ func (a *AnalyzeClusterResource) analyzeResource(analyzer *troubleshootv1beta2.C
 	selected, err := FindResource(analyzer.Kind, analyzer.ClusterScoped, analyzer.Namespace, analyzer.Name, getFileContents)
 	if err != nil {
 		klog.Errorf("failed to find resource: %v", err)
+	}
+	if err != nil || selected == nil {
 		return &AnalyzeResult{
 			Title:   a.Title(),
 			IconKey: "kubernetes_text_analyze",
 			IconURI: "https://troubleshoot.sh/images/analyzer-icons/text-analyze.svg",
 			IsFail:  true,
-			Message: "resource does not exist",
+			Message: fmt.Sprintf("%s %s does not exist", analyzer.Kind, analyzer.Name),
```

Comment:
might be helpful to add the namespace here as well for resources that aren't cluster scoped. otherwise, everything makes sense to me.